cypress/integration/seededFlows/articleFlows/adminButtonArticle.spec.js

```diff
@@ -0,0 +1,88 @@
+describe('Admin button on article - Anonymous user', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.visit('/');
```

Comment:
(just a suggestion) sometime after the pin post tests were first added, we added a hard-coded slug for the test article, so now we could go directly there via `/admin_mcadmin/test-article-slug` instead of finding/clicking the link on line 8

Same with the other tests below where we create articles to then navigate to them - maybe it's enough to just view the test article?

Reply:
The double click does happen to me too! And Julianna. I really have no idea why and I've been debugging it for an hour now.

Will refactor the test :) thanks for the tip!

Reply:
Okay fixed, and updated the test. The double click was because of the InstantClick library.

Reply:
aaah of course it was 🙈  nice find!